week1/project/ecommerce/src/components/drop-list.jsx

```diff
@@ -0,0 +1,31 @@
+import React, {useState} from 'react';
+
+const DropList = ({options, onSelect}) => {
+    const [selectedOption, setSelectedOption] = useState(null);
+    const [isOpen, setIsOpen] = useState(false);
```

Comment:
I like how you are passing functions already as props and using them in child components to change the state in the parent! 👍 

However, you don't need to store this piece of information twice, the option/selected category. `onSelect` is receiving the `option` and in the parent component of App it is setting it as `selectedCategory`. So instead of storing it in the DropList component too you can just pass it from the parent to DropList: 

```suggestion
const DropList = ({options, onSelect, selectedOption}) => {
    const [isOpen, setIsOpen] = useState(false);
```